### PR TITLE
Shubham dalal work

### PR DESCRIPTION
**Pull request by shubhamdalal49**

### CODE REVIEW

File: Shubham_Dalal/JavaScript/Assignment7/Assign_7_Q1.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="Assignment7.css" />
+        <title>Triangle</title>
+        <script>
+            function triangle() {
+                var i,
```
Comment (nileshp1122) [inline Shubham_Dalal/JavaScript/Assignment7/Assign_7_Q1.html]: required in one line

File: Shubham_Dalal/JavaScript/Assignment7/Assign_7_Q3.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="Assignment7.css" />
+        <title>Assignment 7</title>
+        <script>
+            function num() {
+                // debugger
```
Comment (nileshp1122) [inline Shubham_Dalal/JavaScript/Assignment7/Assign_7_Q3.html]: remove comment

File: Shubham_Dalal/JavaScript/Assignment7/Assign_7_Q4.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="Assignment7.css" />
+        <title>Assignment 7</title>
+        <script>
+            function num() {
+                // debugger
```
Comment (nileshp1122) [inline Shubham_Dalal/JavaScript/Assignment7/Assign_7_Q4.html]: remove comment

File: Shubham_Dalal/JavaScript/Assignment7/Assignment7.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
+body {
+    /* background-image: linear-gradient(to right, rgb(62, 22, 75),rgb(60, 23, 86)); */
```
Comment (nileshp1122) [inline Shubham_Dalal/JavaScript/Assignment7/Assignment7.css]: remove comment

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**